integration_test.go

```diff
@@ -344,17 +569,20 @@ func generateData(numRows int) error {
 	}
 
 	//send json marshalled data
-	for i := 0; i < numRows; i++ {
+	for i := 1; i <= numRows; i++ {
 		curr := log_entry{
 			Message: "hello world",
 		}
 		entry, err := json.Marshal(curr)
+		if sendBadRow && ((i % 5) == 0) {
```

Comment:
What is this magic number 5?

Reply:
I wanted to send 2 bad rows (one in the middle of a batch and one at the end) so I chose 5. Do you want me to change this to a constant? Maybe numRows/2?

Reply:
Done.